httphandler.go

```diff
@@ -44,6 +65,85 @@ func (s *MessagingServer) StartServer() error {
 	return nil
 }
 
+func (s *MessagingServer) handleLogIssue(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	// Read in none-file form fields
+	description := strings.TrimSpace(r.FormValue("description"))
+	clientName := strings.TrimSpace(r.FormValue("name"))
+	clientEmail := strings.TrimSpace(r.FormValue("email"))
+	clientPhoneNumber := strings.TrimSpace(r.FormValue("phone"))
+	callback := strings.TrimSpace(r.FormValue("callback"))
+	browserName := strings.TrimSpace(r.FormValue("browser"))
+	browserVersion := strings.TrimSpace(r.FormValue("browserVersion"))
+
+	messageBuffer := new(bytes.Buffer)
+	mpWriter := multipart.NewWriter(messageBuffer)
+
+	for _, files := range r.MultipartForm.File {
+		for _, file := range files {
+			var infile multipart.File
+
+			// Read file data into memory
+			infile, err := file.Open()
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+
+			// Create file field on form to be sent
+			part, err := mpWriter.CreateFormFile(file.Filename, file.Filename)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+
+			// Copy file data to form
+			_, err = io.Copy(part, infile)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+		}
+	}
+
+	mailBody := fmt.Sprintf(helpDeskTemplate, clientName, clientEmail, clientPhoneNumber, callback, browserName, browserVersion, description)
+	mpWriter.WriteField("emailTo", s.Config.HelpDesk)
+	mpWriter.WriteField("subject", "Issue detected in production")
+	mpWriter.WriteField("ishtml", "True")
+	mpWriter.WriteField("replyTo", clientEmail)
+	mpWriter.WriteField("message", mailBody)
+
+	err := mpWriter.Close()
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	mailRequest, err := http.NewRequest("POST", s.Config.EmailUrl+"/sendEmail", messageBuffer)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	mailRequest.Header.Set("Content-Type", mpWriter.FormDataContentType())
+	mailRequest.SetBasicAuth("imqs", s.Config.EmailPassword)
```

Comment:
While we're at it we could also make the email username configurable (`s.Config.EmailUsername`)